ja/tutorials-and-examples/blog/blog.rst

```diff
@@ -37,8 +37,9 @@ CakePHP のダウンロード
 `https://github.com/cakephp/cakephp/tags <https://github.com/cakephp/cakephp/tags>`_
 そして、2.0 の最新リリースをダウンロードします。
 
-または、 `git <http://git-scm.com/>`_ を使ってレポジトリを clone することもできます。
-``git clone git://github.com/cakephp/cakephp.git``
+または、 `git <http://git-scm.com/>`_ を使ってレポジトリを clone することもできます。 ::
+
+    git clone git://github.com/cakephp/cakephp.git
```

Comment:
This command should be:
<code>git clone <strong>-b 2.x</strong> git://github.com/cakephp/cakephp.git</code>